core/src/main/java/org/ode4j/ode/internal/joints/DxJointLimitMotor.java

```diff
@@ -399,26 +386,33 @@ int addTwoPointLimot( DxJoint joint, double fps,
 			Info2Descr info, int row,
 			DVector3C ax1, DVector3C pt1, DVector3C pt2 )
 	{
-		int srow = row * info.rowskip();
 
 		// if the joint is powered, or has joint limits, add in the extra row
 		boolean powered = fmax > 0;
+		DVector3 j1l = new DVector3();

```

Comment:
Would it make sense to simply write `DVector3C j1l = ax1;' ? That would avoid the copying and allocating additional vectors. At least for j1l, it doesn't seem to be modified anywhere (and ax1 is immutable anyway).